contracts/LosslessSecurityOracle.sol

```diff
@@ -132,11 +133,8 @@ contract LosslessSecurityOracle is
         override
         returns (uint8)
     {
-        if (!getIsSubscribed(msg.sender)) {
-            return 0;
-        } else {
-            return riskScores[_address];
-        }
+        require(getIsSubscribed(msg.sender), "LSS: Not subscribed");
```

Comment:
CVF-10 is not valid. We intentionally want to return 0 so that protocol that's using the oracle does not need to worry about handling reverts and once the subscription is expired just stops seeing the real security scores.  This approach would brick the defi protocol every time the subscription is expired.

Reply:
Reverted on commit 058d5de